API.md

```diff
@@ -1643,6 +1643,47 @@ this.player.seek(120, 50); // Seek to 2 minutes with +/- 50 milliseconds accurac
 
 Platforms: iOS
 
+### setPlayerStatus
+`setPlayerStatus(shouldPlay: boolean): Promise<void>`
+
+Play/Pause the video.
+
+Example:
+```
+this.player.setPlayerStatus(true); // Play the video
+this.player.setPlayerStatus(false); // Pause the video
+```
+
+platforms: Android, iOS
+
+### pause
+`pause(): Promise<void>`
```

Comment:
do it really need to return a Promise as there is no result ?

Reply:
Yeah, it return promise to let know user that this function is async

Reply:
Ok, but the question is why is it needed to be async ?
Here you bring more complexity to something which can be straight forward.
if user really need to delay it he use its own promise.
There is nothing interesting with promise: I mean waiting for the promise result doesn't bring interesting information to the user (there is be another event indicating player is paused).

That said I would be curious to have a performance comparaison  between 3 solutions:
- using the pause property
- using direct call (the solution I propose)
- using direct call embedded in promise
- using async call (the solution you implement)

Anyway I agree to keep it like this.
Thank you